pkg/pods/hoist_launchable.go

```diff
@@ -53,9 +53,14 @@ func (hoistLaunchable *HoistLaunchable) Halt(serviceBuilder *runit.ServiceBuilde
 }
 
 func (hoistLaunchable *HoistLaunchable) Launch(serviceBuilder *runit.ServiceBuilder, sv *runit.SV) error {
+	_, err := hoistLaunchable.Disable()
```

Comment:
This should be extracted to `orchestrate.go` as `pod.Disable(oldManifest)`. As written this would disable the new manifest, as well as assuming that there _is_ a previous install to disable.